sampler/stories/default/Input.js

```diff
@@ -17,16 +17,19 @@ storiesOf('Agate', module)
 		'Input',
 		() => (
 			<Input
-				onChange={action('onChange')}
+				defaultValue={text('defaultValue', Config, '')}
```

Comment:
Let's remove the `defaultValue` altogether. I'm not sure why it was here originally but it works fine without it and is an exception we don't need to maintain.